docs/source/reference.rst

```diff
@@ -47,6 +47,42 @@ Mobjects
    ~mobject.types.point_cloud_mobject
    ~mobject.types.vectorized_mobject
 
+Mobjects Inheritance Graph
```

Comment:
```suggestion
**Mobject** Inheritance Graph
```
I think it would look a bit better with only singular.
And to make it a bit more visible, I would add bold text